.github/auto-assignees.yaml

```diff
@@ -1,33 +1,34 @@
-# This assigns a PR to its author
-addAssignees: true
+addAssignees: false
 
-addReviewers: false
+addReviewers: true
 
-reviewers:
-  - holyhope
-  - sguyennet
-  - jMonsinjon
-  - steven-zou
-  - mhurtrel
-  - heww
-  - bitsf
-  - wangcanfengxs
-  - Ghostbaby
-  - soulseen
-  - chlins
-  - XavierDuthil
-  - cndoit18
-  - lubronzhan
-  - yxxhero
-  - wy65701436
-  - MarcelMue
-  - thcdrt
+useReviewGroups: true
 
-options:
-  ignore_draft: true
-  ignored_keywords:
-    - WIP
-    - wip
-    - DO NOT MERGE
-  enable_group_assignment: true
-  number_of_reviewers: 2
+reviewGroups:
+  active:
+    - MarcelMue
+    - thcdrt
+  general:
+    - holyhope
+    - sguyennet
+    - jMonsinjon
+    - steven-zou
+    - mhurtrel
+    - heww
+    - bitsf
+    - wangcanfengxs
+    - Ghostbaby
+    - soulseen
+    - chlins
+    - XavierDuthil
+    - cndoit18
+    - lubronzhan
+    - yxxhero
+    - wy65701436
+
+skipKeywords:
+  - WIP
+  - wip
+  - DO NOT MERGE
+
+numberOfReviewers: 2
```

Comment:
```suggestion
numberOfReviewers: 1
```

?

Reply:
The 2 is because:
I added two groups of reviewers - one with the two of us and one with all the other maintainers. The 2 here will cause automation to (hopefully) always assign one of us and one of all the other maintainers as reviewers.

Reply:
Ho ok I thought it was defining the number of minimal approves, mb